common/logisticspipes/proxy/side/ClientProxy.java

```diff
@@ -244,12 +248,21 @@ public void openFluidSelectGui(final int slotId) {
 		}
 	}
 
+	@Override
+	public void openGuiFromItem(int guiId, EnumHand hand) {
+		Minecraft mc = Minecraft.getMinecraft();
+		EntityPlayer player = mc.player;
+		World world = mc.world;
+		BlockPos pos = player.getPosition();
+		player.openGui(LogisticsPipes.instance, guiId, world, hand.ordinal(), pos.getY(), pos.getZ());
```

Comment:
What about `mc.displayGuiScreen(new GuiGuideBook(hand))`? You're overcomplicating this way too much here.

Reply:
Corrected